resources/views/layouts/default/flights/simbrief_form.blade.php

```diff
@@ -1,269 +1,411 @@
-                    </select></td>
-                </tr>
-                <tr>
-                  <td>Reserve Fuel:</td>
-                  <td><select name="resvrule">
-                      <option value="auto">AUTO</option>
-                      <option value="0">0 MIN</option>
-                      <option value="15">15 MIN</option>
-                      <option value="30">30 MIN</option>
-                      <option value="45" selected>45 MIN</option>
-                      <option value="60">60 MIN</option>
-                      <option value="75">75 MIN</option>
-                      <option value="90">90 MIN</option>
-                    </select></td>
-                </tr>
-                <tr>
-                  <td>Detailed Navlog:</td>
-                  <td><input type="hidden" name="navlog" value="0"><input type="checkbox" name="navlog" value="1"
-                                                                          checked>
-                  </td>
-                </tr>
-                <tr>
-                  <td>ETOPS Planning:</td>
-                  <td><input type="hidden" name="etops" value="0"><input type="checkbox" name="etops" value="1"></td>
-                </tr>
-                <tr>
-                  <td>Plan Stepclimbs:</td>
-                  <td><input type="hidden" name="stepclimbs" value="0"><input type="checkbox" name="stepclimbs"
-                                                                              value="1"
-                                                                              checked>
-                  </td>
-                </tr>
-                <tr>
-                  <td>Runway Analysis:</td>
-                  <td><input type="hidden" name="tlr" value="0"><input type="checkbox" name="tlr" value="1" checked>
-                  </td>
-                </tr>
-                <tr>
-                  <td>Include NOTAMS:</td>
-                  <td><input type="hidden" name="notams" value="0"><input type="checkbox" name="notams" value="1"
-                                                                          checked>
-                  </td>
-                </tr>
-                <tr>
-                  <td>FIR NOTAMS:</td>
-                  <td><input type="hidden" name="firnot" value="0"><input type="checkbox" name="firnot" value="1"></td>
-                </tr>
-                <tr>
-                  <td>Flight Maps:</td>
-                  <td><select name="maps">
-                      <option value="detail">Detailed</option>
-                      <option value="simple">Simple</option>
-                      <option value="none">None</option>
-                    </select></td>
-                </tr>
-              </table>
-            </div>
-          </div>
-        </div>
+            @if($totalgenload > 0)
+              @php
+                $loaddisttxt =  "Load Distribution ";
+                $loaddist = implode(' ', array_map(
```

Comment:
What is this doing? What exactly is being mapped?

Reply:
It is mapping the loadarray array , which is created above before generating loads for each fare type assigned to the aircraft's subfleet , it holds the farecode and the generated random load for that fare ... Then used here to create the load distribution message sent to simbrief as manualrmk

Reply:
I think we have no more issues on this, if you do not want it in the dev we can remove it too.

Not the loadarray of course it is needed for the calculations itself but we can remove the manualrmk section too as we did for the info text.

Personally I like the idea keeping it, maybe some acars developer reads the dispatch remarks from the simbrief.xml and uses it somewhere ;)